test/cypress/integration/group-chat-scrolling.spec.js

```diff
@@ -30,7 +30,7 @@ describe('Send/edit/remove messages & add/remove emoticons inside group chat', (
 
   function sendMessage (message) {
     cy.getByDT('messageInputWrapper').within(() => {
-      cy.get('textarea').clear().type(`${message}{enter}`)
+      cy.get('textarea').clear().type(`${message}{enter}`, { delay: 0 })
```

Comment:
`delay: 0` is an option to give no delay between key-strokes here ([reference](https://docs.cypress.io/api/commands/type#Arguments)).  `sendMessage` here is used for an action that is repeated many times (e.g. `user1 sends 15 messages` etc), so thought I would want to make it a little faster.